firewood/tests/db.rs

```diff
@@ -99,11 +101,18 @@ fn test_revisions() {
             .collect();
         key
     };
+
+    let mut tmpdir: PathBuf = std::env::var_os("CARGO_TARGET_DIRY")
+        .unwrap_or(temp_dir().into())
+        .into();
+    tmpdir.push("/tmp/testdb");
```

Comment:
Is it time for `crate::test_utils`? Maybe in a follow up?

Reply:
Yes, follow up